src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -8,21 +8,23 @@
 
     public static final String COMMAND_WORD = "help";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
-            + "Example: " + COMMAND_WORD;
+    public static final String MESSAGE_USAGE =
+            COMMAND_WORD + ": Shows program usage instructions.\n"
+                    + "Example: " + COMMAND_WORD;
```

Comment:
This doesn't seem to follow the coding standards. You can move the second line to the first line as long as it's not reaching the limit.

Reply:
I think it got auto wrapped due to the limit I set which is at 100 chars. Will look into it again. Thanks!